emoji/src/main/java/com/vanniktech/emoji/EmojiUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.vanniktech.emoji;
+
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class EmojiUtils {
```

Comment:
I think the `EmojiManager` should provide this functionality and not another class.

@vanniktech Maybe we should make all of the `EmojiManager` public now as with this included changes it is quite usefull for other things apart from only managing `TextViews` and `EditTexts`.